devkit/states/git/pr_git_aliases

```diff
@@ -0,0 +1,53 @@
+# vim: filetype=sh
+
+# Source from .bashrc or .zshrc
+
+alias gad="git add"
```

Comment:
Do we want aliases for any other utilities?

Reply:
possibly for commands that we install